packages/next/taskfile.js

```diff
@@ -72,8 +72,7 @@ const externals = {
   chokidar: 'chokidar',
   'jest-worker': 'jest-worker',
 
-  'terser-webpack-plugin':
-    'next/dist/build/webpack/plugins/terser-webpack-plugin/src',
+  'terser-webpack-plugin': 'terser-webpack-plugin',
```

Comment:
To resolve the following error:

```sh
> Build error occurred
TypeError: E is not a constructor
    at Object.apply (/.../node_modules/next/dist/compiled/webpack/bundle5.js:28:595932)
```

Compiled `next/dist/compiled/webpack/bundle5.js` also.